### ISSUE TITLE
Chat Over Imap

### ISSUE THREAD

**Issue by ScottPeterJohnson** [ScottPeterJohnson/purelymail-issues#12]
(This issue was imported from Gitea)
**Scott** on February 28, 2019:
Chat Over Imap is about what it sounds like, a hacky way to turn email into chat. It's interesting enough that we might implement it as an optional feature. Care should be taken to not clutter the basic email experience, however, which may require developing a plugins system.
https://www.coi-dev.org/

**Comment by ScottPeterJohnson** [comment]
Comment by **alex-7fpWKwwsGRSusbQB** on December 30, 2019:
Hey there.

For what it's worth, COI is related to / a fork? of [Delta.Chat](https://delta.chat) at least based on [this HN thread](https://news.ycombinator.com/item?id=19216346). 

I like the idea of this very much. If I were the one making decisions, I would look into Delta Chat itself, not the (prettier, granted) repackaged version that may come with incompatibility at some point if possible future investors start twisting thumb-screws to make the numbers look better.

**Comment by sysfu** [comment]
There might need to be some exceptions or special whitelisting for email accounts configured to use DeltaChat. I recently setup a trial purelymail account to use with DeltaChat, and quickly got a permanent SMTP block. I assume this is because the rapid fire messages are tripping the anti-spam filters.

**Comment by paulo-roger** [comment]
> There might need to be some exceptions or special whitelisting for email accounts configured to use DeltaChat. I recently setup a trial purelymail account to use with DeltaChat, and quickly got a permanent SMTP block. I assume this is because the rapid fire messages are tripping the anti-spam filters.

yes, with disroot for example we need to tell the staff we are gonna use that inbox with Delta Chat so they can adjust some rules.

**Comment by femme-theory23** [comment]
I don't believe that that should have been flagged, other than just a timeout notice. (At least not with the current setup as I see it). However, I doubt that we could easily offer it without needing to massively change the rate limits and billing systems, as this would want to use up a lot of sending rate limits as I see it, where the charges would quickly build up, just for a single user, let alone a multi user account.
